classes/Page/Page.mjs

```diff
@@ -0,0 +1,44 @@
+export class Page {
+    constructor(page = {}) {
+        this.page = page
+        if(!this.page.id) {
+            this.page.id = Date.now().toString();  // ticket to replace with MongoDB ObjectID()
+        }
+    }
+    
+    get id() {
+        return this.page.id
+    }
+
+    set id(id) {
+        this.page.id = id
+    }
+
+
+    asCanvas() {
+        return {
+            type: 'Annotation',
```

Comment:
Look at the "Canvas" object for basic shape of a Canvas at https://iiif.io/api/presentation/3.0/

Reply:
Updated on this , I had gone through the document and added the required properties.